Makefile

```diff
@@ -6,4 +6,14 @@ changelog:
 release:
 	semtag final -s minor
 
+.PHONY: test
 test:
+	pre-commit run
+
+.PHONY: test-slow
+test-single-region: test
+	cd ./test/ &&	python3 test.py > test-single-region.txt 2>&1
```

Comment:
Writing test output to disk is unexpected, if there's a good reason to do this make sure it's included in `.gitignore`

Reply:
removed writing to disk